src/gtest/test_joinsplit.cpp

```diff
@@ -22,6 +22,39 @@ using namespace libzcash;
 
 extern ZCJoinSplit* params;
 
+typedef std::vector<JSDescription> SproutProofs;
```

Comment:
This will always be length 2, so make this a `std::array<JSDescription, 2>` (same as the inputs to `makeSproutProofs()`).